util/config.go

```diff
@@ -73,6 +74,7 @@ func NewConfig(opts ...ConfigOption) (rc *Config, err error) {
 		DisabledResourceTypes: []string{},
 		ListenAddress:         ":8080",
 		LocalAddress:          "",
+		LogLevel:              "FATAL",
```

Comment:
would it be possible to add a few helpers like `WithLogLevel(l string) and `WithLogWriter(w io.Writer)` or maybe WithLogger(c *Config)` to allow the case of embedding the code to be able to modify these new config items programatically?

Reply:
Indeed, I also think this could be a very nice to have in the future but I will need some more time to work on this

Could it be possible to merge this as is for now and I'll work on a next more complete PR to address this

Reply:
I am ok with merging as is. I am the main user of the embedded behaviours (that this enables) so can also propose a follow up PR for that